Changing parent sampled flag breaks distributed traces. (#7265)
getsentry/sentry-docs: src/platforms/common/configuration/sampling.mdx

</PlatformSection>

> If the transaction currently being created is one of those subsequent transactions (in other words, if it has a parent transaction), the upstream (parent) sampling decision will always be included in the sampling context data, so that your <PlatformIdentifier name="traces-sampler" /> can choose whether and when to inherit that decision. (In most cases, inheritance is the right choice, to avoid breaking distributed traces.)

Reviewer: ```suggestion
If the transaction currently being created is one of those subsequent transactions (in other words, if it has a parent transaction), the upstream (parent) sampling decision will be included in the sampling context data. Your <PlatformIdentifier name="traces-sampler" /> can use this information to choose whether to inherit that decision. In most cases, inheritance is the right choice, to avoid breaking distributed traces.
```

Again, can we say what property users can use to see the parent sampling decision?

When there's the potential for more than one of these to come into play, the following precedence rules apply:

1. If a sampling decision is passed to <PlatformIdentifier name="start-transaction" />, that decision will be used, overriding everything else.

> 1. If <PlatformIdentifier name="traces-sampler" /> is defined, its decision will be used. It can choose to keep or ignore any parent sampling decision, or use the sampling context data to make its own decision or choose a sample rate for the transaction. (We adive against overriding the parent sampling decision, because it will break distirbuted traces.)

Reviewer 2: ```suggestion
1. If <PlatformIdentifier name="traces-sampler" /> is defined, its decision will be used. It can choose to keep or ignore any parent sampling decision, or use the sampling context data to make its own decision, or choose a sample rate for the transaction. (We advise against overriding the parent sampling decision, because it will break distributed traces.)
```

Reviewer: tweaks to @sentrivana 's suggestion
